scripts/release/check_commit_metadata.sh

```diff
@@ -78,92 +160,143 @@ main() {
 			--base main \
 			--state merged \
 			--limit 10000 \
-			--search "merged:>=$from_commit_date" \
+			--search "merged:>=${lookback_date}" \
 			--json mergeCommit,labels,author \
 			--jq '.[] | "\( .mergeCommit.oid ) author:\( .author.login ) labels:\(["label:\( .labels[].name )"] | join(" "))"'
 	)"
 
 	declare -A authors labels
-	if [[ -n $pr_list_out ]]; then
-		mapfile -t pr_metadata_raw <<<"$pr_list_out"
+	if [[ -n ${pr_list_out} ]]; then
+		mapfile -t pr_metadata_raw <<<"${pr_list_out}"
 
 		for entry in "${pr_metadata_raw[@]}"; do
 			commit_sha_long=${entry%% *}
 			commit_author=${entry#* author:}
 			commit_author=${commit_author%% *}
-			authors[$commit_sha_long]=$commit_author
+			authors[${commit_sha_long}]=${commit_author}
 			all_labels=${entry#* labels:}
-			labels[$commit_sha_long]=$all_labels
+			labels[${commit_sha_long}]=${all_labels}
 		done
 	fi
 
 	for commit in "${commits[@]}"; do
-		mapfile -d ' ' -t parts <<<"$commit"
-		commit_sha_short=${parts[0]}
-		commit_sha_long=${parts[1]}
-		commit_prefix=${parts[2]}
+		mapfile -d ' ' -t parts <<<"${commit}"
+		left_right=${parts[0]}
```

Comment:
not sure I understand this var `left_right`, maybe leave a comment or add example here?

Reply:
It's git terminology (see above command run with `--left-right`), git man page explains this best but I can add a comment to refer there.